integration-libs/cdc/core/auth/services/user-authentication/cdc-login.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from "@angular/core";
+import { LoginFormComponentService } from "feature-libs/user/account/components/login-form";
+import { CdcJsService } from "integration-libs/cdc/root";
+import { AuthService } from "projects/core/src/auth";
+import { GlobalMessageService } from "projects/core/src/global-message";
+import { WindowRef } from "projects/core/src/window";
+import { BehaviorSubject } from "rxjs";
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CdcLoginComponentService extends LoginFormComponentService {
+  constructor(
+    protected auth: AuthService,
+    protected globalMessage: GlobalMessageService,
+    protected winRef: WindowRef,
+    protected cdcJsService: CdcJsService
+  ) {
+    super(auth, globalMessage, winRef);
+  }
+
+  protected busy$ = new BehaviorSubject(false);
```

Comment:
Move busy before constructor and type it ` protected busy$: BehaviorSubject<boolean> = new BehaviorSubject(false);`